src/common/constants/constants.ts

```diff
@@ -10,7 +10,7 @@ export const enumMappings: { [key in Role]: EnumMapping } & {
   [key in Genre]: EnumMapping
 } = {
   director_of_photography: {
-    label: 'Director of Photography',
+    label: 'Director Of Photography',
```

Comment:
This is _very_ silly but grammatically the "Of" should be lowercase

Reply:
Not silly at all!

I capitalized it because of this part of the wireframe:
![image](https://github.com/mwkwsd/sensensomething/assets/29478554/99f824a9-af1d-4b5b-b1c4-9afb002f1041)

However, I can see in more recent designs, she's used a lowercase o

```suggestion
    label: 'Director of Photography',
```